prime-router/metadata/hl7_mapping/ORU_R01/base/xad-extended-address.yml

```diff
@@ -1,36 +1,79 @@
 name: ORU-R01-xad-extended-address
+constants:
+  # if orderIndex is provided, we use the "-ordered" elements to support multiple entries
+  fieldPath: (%{hl7AddressField})(%{orderIndex})
 elements:
   - name: street-address
     condition: '%resource.line.count() > 0'
     value: [ '%resource.line[0]' ]
     hl7Spec: [ '%{hl7AddressField}-1' ]
 
+  - name: street-address-ordered
```

Comment:
There seem to be duplicate fields here. Line 5 and 11.  Some more below

Reply:
fixed